project_collaborator/security/rules.xml

```diff
@@ -0,0 +1,13 @@
+<odoo noupdate="1">
+    <!-- Project: Collaborators can see assigned projects -->
```

Comment:
Is that enough to see projects for the collaborators with `base.group_user`? Project app is only visible for project users. Or am I miss something?
https://github.com/odoo/odoo/blob/059b19c59a9222b676be5246eb94d680cf6bf27b/addons/project/views/project_views.xml#L5-L9